#control_mode:
#    wheel_joint_1: 26
#    wheel_joint_2: 26
#    wheel_joint_3: 26
#    wheel_joint_4: 26

stiffness:
    hip_roll_1: 1500
    hip_pitch_1: 1500
    knee_pitch_1: 1500
    hip_roll_2: 1500
    hip_pitch_2: 1500
    knee_pitch_2: 1500
    hip_roll_3: 1500
    hip_pitch_3: 1500
    knee_pitch_3: 1500
    hip_roll_4: 1500
    hip_pitch_4: 1500
    knee_pitch_4: 1500
    shoulder_yaw_1: 100
    shoulder_pitch_1: 100
    elbow_pitch_1: 100
    wrist_pitch_1: 50
    wrist_yaw_1: 50
    shoulder_yaw_2: 100
    shoulder_pitch_2: 100
    elbow_pitch_2: 100
    wrist_pitch_2: 50
    wrist_yaw_2: 50
    wheel_joint_1: 0
    wheel_joint_2: 0
    wheel_joint_3: 0
    wheel_joint_4: 0

#stiffness:
#    hip_roll_1: 0
#    hip_pitch_1: 0
#    knee_pitch_1: 0
#    hip_roll_2: 0
#    hip_pitch_2: 0
#    knee_pitch_2: 0
#    hip_roll_3: 0
#    hip_pitch_3: 0
#    knee_pitch_3: 0
#    hip_roll_4: 0
#    hip_pitch_4: 0
#    knee_pitch_4: 0
#    shoulder_yaw_1: 0
#    shoulder_pitch_1: 0
#    elbow_pitch_1: 0
#    wrist_pitch_1: 0
#    wrist_yaw_1: 0
#    shoulder_yaw_2: 0
#    shoulder_pitch_2: 0
#    elbow_pitch_2: 0
#    wrist_pitch_2: 0
#    wrist_yaw_2: 0
#    wheel_joint_1: 0
#    wheel_joint_2: 0
#    wheel_joint_3: 0
#    wheel_joint_4: 0

damping:
    hip_roll_1: 25
    hip_pitch_1: 25
    knee_pitch_1: 25
    hip_roll_2: 25
    hip_pitch_2: 25
    knee_pitch_2: 25
    hip_roll_3: 25
    hip_pitch_3: 25
    knee_pitch_3: 25
    hip_roll_4: 25
    hip_pitch_4: 25
    knee_pitch_4: 25
    shoulder_yaw_1: 5
    shoulder_pitch: 5
    elbow_pitch_1: 5
    wrist_pitch_1: 5
    wrist_yaw_1: 5
    shoulder_yaw_2: 5
    shoulder_pitch: 5
    elbow_pitch_2: 5
    wrist_pitch_2: 5
    wrist_yaw_2: 5
    wheel_joint_1: 10
    wheel_joint_2: 10
    wheel_joint_3: 10
    wheel_joint_4: 10

#damping:
#    hip_roll_1: 0
#    hip_pitch_1: 0
#    knee_pitch_1: 0
#    hip_roll_2: 0
#    hip_pitch_2: 0
#    knee_pitch_2: 0
#    hip_roll_3: 0
#    hip_pitch_3: 0
#    knee_pitch_3: 0
#    hip_roll_4: 0
#    hip_pitch_4: 0
#    knee_pitch_4: 0
#    shoulder_yaw_1: 0
#    shoulder_pitch: 0
#    elbow_pitch_1: 0
#    wrist_pitch_1: 0
#    wrist_yaw_1: 0
#    shoulder_yaw_2: 0
#    shoulder_pitch: 0
#    elbow_pitch_2: 0
#    wrist_pitch_2: 0
#    wrist_yaw_2: 0
#    wheel_joint_1: 10
#    wheel_joint_2: 10
#    wheel_joint_3: 10
#    wheel_joint_4: 10
